.circleci/config.yml

```diff
@@ -82,7 +82,7 @@ commands:
     steps:
       - run:
           name: Run Integration Tests
-          command: go test -tags=integration -v ./test/integration/...
+          command: go test -tags=integration -timeout=30m -v ./test/integration/...
```

Comment:
Can we start changing it to 15?

Reply:
We can try 15 of course, because the amount of time it would need in CircleCI won't
be the same we observe in our own environment. If it does not work, then we can tune
later.

Reply:
Let me know if you prefer to try with 15 first and I will update the PR.

Reply:
yup, it is not a requirement, since we really do not know if 15 is ok but, 30 could be wrong also, In that spirit, I prefer to increase as less as possible.

Reply:
done.